lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -1623,13 +1623,16 @@ template <typename T> void ConfigBuilder::buildPsRegConfig(ShaderStage shaderSta
                              fragmentMode.postDepthCoverage);
   }
 
+  // NOTE: MRT0's alpha may need to be copied to MRTZ's alpha on GFX11+, we need update the depth export format
+  // accordinately.
   unsigned depthExpFmt = EXP_FORMAT_ZERO;
   if (builtInUsage.sampleMask)
     depthExpFmt = EXP_FORMAT_32_ABGR;
   else if (builtInUsage.fragStencilRef)
-    depthExpFmt = EXP_FORMAT_32_GR;
+    depthExpFmt =
+        m_pipelineState->getTargetInfo().getGfxIpVersion().major >= 11 ? EXP_FORMAT_32_ABGR : EXP_FORMAT_32_GR;
   else if (builtInUsage.fragDepth)
-    depthExpFmt = EXP_FORMAT_32_R;
+    depthExpFmt = m_pipelineState->getTargetInfo().getGfxIpVersion().major >= 11 ? EXP_FORMAT_32_AR : EXP_FORMAT_32_R;
```

Comment:
The wider export formats should only be used when alpha-to-mask is enabled.

Reply:
Oops. I made a wrong rebase. There code should be removed.